src/components/ButtonComponents/OperatorButtons/OperatorButton.js

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 
-const OperatorButton = () => {
+export const OperatorButton = (operatorKey) => {
```

Comment:
Same thing here. This argument is always props. It contains the keys from the 'attributes' we added to the component when we render it.